locations/spiders/virginia_health_inspection_blue_ridge.py

```diff
@@ -0,0 +1,55 @@
+from scrapy import Spider
+from scrapy.http import JsonRequest
+
+from locations.dict_parser import DictParser
+
+
+class VirginiaHealthInspectionBlueRidgeSpider(Spider):
+    name = "virginia_health_inspection_blue_ridge"
+    item_attributes = {}
+    start_urls = ["https://inspections.myhealthdepartment.com"]
+
+    def start_requests(self):
+        for url in self.start_urls:
+            yield JsonRequest(
+                url=url,
+                callback=self.parse_location_list,
+                data={
+                    "data": {
+                        "path": "va-blue-ridge",
+                        "programName": "",
+                        # TODO: make the date dynamic
+                        "filters": {"date": "2024-01-01 to 2024-09-27", "purpose": "", "city": "", "zip": ""},
+                        "start": 0,
+                        "count": 2000,
+                        "searchQueryOverride": None,
+                        "searchStr": "",
+                        "lat": 0,
+                        "lng": 0,
+                    },
+                    "task": "searchInspections",
+                },
+            )
+
+    def parse_location_list(self, response):
+        for inspection in response.json():
+            item = DictParser.parse(inspection)
+
+            item["ref"] = inspection["inspectionID"]
+            item["name"] = inspection["establishmentName"]
+
+            item["extras"] = ({"inspectionDate": inspection["inspectionDate"]},)
```

Comment:
Not a set

```suggestion
            item["extras"] = {"inspectionDate": inspection["inspectionDate"]}
```

Also would it be worth using a more standard `check_date` or `survey:date`?

Reply:
Actually don't re init extras
```suggestion
            item["extras"]["inspectionDate"] = inspection["inspectionDate"]
```

Reply:
> Also would it be worth using a more standard check_date or survey:date?
I wasn't sure if this would be well received, since it's not actually me checking the location, but a third party. Open to discussion on changing it though!

Reply:
I would suggest it's close enough, but no strong preference.